### PR TITLE
[Feature] Improve cross chain TX history and status - Part 1

### PR DESCRIPTION
**Pull request by yuripramos**
This PR is part one of a [list of improvements/extensions over our bridge tx's ](https://swing-world.monday.com/boards/1924327886/pulses/1938130690)

- [ ] Connect transfer to approval. 
- [x] [Fix transaction history on explorer](https://swing-world.monday.com/boards/1924327886/pulses/2234925210).
- [x] [Make a generic transaction history from SDK's within our tx history modal](https://swing-world.monday.com/boards/1924327886/pulses/2234929118)
- [ ] Fix to enable notifications across tx history at the right top. 
- [x] [Min allowed for cBridge SDK to make tx](https://swing-world.monday.com/boards/1924327886/pulses/2234927943)

### CODE REVIEW

File: src/client/js/utils/txBridgeManager.js [hunk 1]
```diff
@@ -241,10 +264,94 @@ export default {
       return false;
     }
 
-    return tx.bridge === 'connext';
+    return tx.bridge === 'connext' || tx.bridge === 'cbridge';
```
Comment (chillvybz) [inline src/client/js/utils/txBridgeManager.js:267]: Is there a two step process for cbridge? i didnt know that.

File: src/client/js/utils/txBridgeManager.js [hunk 2]
```diff
@@ -241,10 +264,94 @@ export default {
...
+    const nxtpQueue = Nxtp.getAllHistoricalTxs();
+    const cBridgeQueue = await CBridgeUtils.getTxHistory();
+
+    this.buildGenericTxHistory(nxtpQueue, 'connext');
+    this.buildGenericTxHistory(cBridgeQueue, 'cbridge');
+
+    return this._genericTxHistory;
+  },
+
+  buildGenericTxHistory(bridgeTxHistory, bridge) {
```
Comment (chillvybz) [inline src/client/js/utils/txBridgeManager.js:286]: Good stuff

File: src/client/js/utils/txBridgeManager.js (hunk 1 above)
Comment (yuripramos) [reply src/client/js/utils/txBridgeManager.js:267]: Yep, considering that cbridge needs to poll through `/v2/getTransferStatus` to wait for the destination chain url and other props, then yes.

**Comment by yuripramos** [comment]
hey @chillvybz I've fixed the conflicts and fixed minor stuff. 
Please review it at your convenience and carefully, since we (still) don't have some build checks over our environment btw this is on my radar to develop

File: src/client/js/utils/txBridgeManager.js (hunk 1 above)
Comment (chillvybz) [reply src/client/js/utils/txBridgeManager.js:267]: Okay but does it require an explicit user action twice?
That's what the goal of the two-step was.

Comment (yuripramos) [reply src/client/js/utils/txBridgeManager.js:267]: not explicit, only a refund will require that.

But If we don't reuse our current approach for 2 steps, there will be a lot of new code exclusively for `cbridge` to extract the correct `tx_hash` for the destination chain. And using practically the same strategy that we have for 2 steps, so "unnecessary" boilerplate

**Pull request merged**